<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ModeloItem</class>
 <widget class="QDialog" name="ModeloItem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Modelo</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/rs.qrc" >:/png/buttons/16/frameprint.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Itens</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Cor de Fundo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Cor Alternativa</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>16</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="leFundo" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>16</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="leFundoAlt" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QTextEdit" name="teItems" >
        <property name="font" >
         <font>
          <family>Courier</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="lineWrapMode" >
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="acceptRichText" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Corpo</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QTextEdit" name="teBody" >
        <property name="font" >
         <font>
          <family>Courier</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="lineWrapMode" >
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="acceptRichText" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_success.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancelar</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_fail.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leModelo" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>leModelo</tabstop>
  <tabstop>teBody</tabstop>
  <tabstop>leFundo</tabstop>
  <tabstop>leFundoAlt</tabstop>
  <tabstop>teItems</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../res/rs.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ModeloItem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
